packages/core/src/components/spinner/_spinner.scss

```diff
@@ -29,18 +27,21 @@ Styleguide spinner
   to   { transform: rotate(360deg); }
 }
 
-// see https://css-tricks.com/scale-svg/
-// how to Scale SVG to Fit the Available Width (and adjust the height to match) Option 4
-
 .#{$ns}-spinner {
   @include spinner-size($spinner-width-factor, $spinner-stroke-width, $spinner-speed);
+  // allow paths to overflow container -- critical for edges of circles!
+  overflow: visible;
+  vertical-align: top;
```

Comment:
is vertical align necessary? is this going to mess with existing uses?

Reply:
hmm good thought. i'll change to `middle`

Reply:
is it even needed?